src/routes/expenses.routes.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const router = express.Router();
+
+const createRouter = ({ userService, expensesService }) => {
```

Comment:
1. You can directly import your services into this file 
2. It is enough you need to export only the router variable from this file